services/appflowy-worker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM lukemathwalker/cargo-chef:latest-rust-1.77 as chef
+
+# Set the initial working directory
+WORKDIR /app
+RUN apt update && apt install lld clang -y
+
+FROM chef as planner
+COPY . .
+
+# Compute a lock-like file for our project
+RUN cargo chef prepare --recipe-path recipe.json
+
+FROM chef as builder
+
+# Update package lists and install protobuf-compiler along with other build dependencies
+RUN apt update && apt install -y protobuf-compiler lld clang
+
+COPY --from=planner /app/recipe.json recipe.json
+# Build our project dependencies
+RUN cargo chef cook --release --recipe-path recipe.json
+
+COPY . .
+
+WORKDIR /app/services/appflowy-worker
+RUN cargo build --release --bin appflowy_worker
+
+FROM debian:bookworm-slim AS runtime
+WORKDIR /app/services/appflowy-worker
+RUN apt-get update -y \
+    && apt-get install -y --no-install-recommends openssl \
+    && apt-get install protobuf-compiler -y \
```

Comment:
We don't have to install protobuf compiler in run time.

Reply:
`lib/database` crate depends on` lib/tonic-proto` create that needs `proto`

Reply:
Yes, we do require protobuf during build process. But in debian:bookworm-slim , where we are running the compiled binary, we don't require the compiler, it is only required for the build process.